src/block-components/helpers/borders/style.js

```diff
@@ -33,6 +33,58 @@ const Styles = props => {
 				hover="all"
```

Comment:
@bfintal left this become the attribute only changes when the block was clicked

Reply:
@prconcepcion please add a comment about this

Also add a comment in the attribute list because you are going to leave `borderRadius` there